### ISSUE TITLE
Clickable link to database project contains erranous square bracket at the end

### ISSUE THREAD

**Issue by maccer83** [microsoft/azuredatastudio#15047]
Issue Type: <b>Bug</b>

When building a sql database project and getting error messages in the console, you can usually click the links. But the link to the database project itself contains a closing bracket at the end (see image where the "]" after ".sqlproj]" is underlined and thus included in the link. 

![Square bracket at end of clickable link](https://user-images.githubusercontent.com/12064302/114026116-2b014b00-9876-11eb-8741-8dc838ff8ae1.png)

When clicking this link, Azure Data Studio cannot find the file and wants me to create it.

![Unable to open file](https://user-images.githubusercontent.com/12064302/114026425-83d0e380-9876-11eb-9e0a-fe995843dc0e.png)

Extension version: 0.7.1
Azure Data Studio version: azuredatastudio 1.27.0 (765c7064d46253d88393e0e1ff78874b6fe8fa64, 2021-03-16T21:18:54.187Z)
OS version: Darwin x64 19.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz (8 x 2500)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (0.12GB free)|
|Process Argv||
|Screen Reader|no|
|VM|25%|
</details>

**Comment by kisantia** [comment]
duplicate of #13955

**Comment by kisantia** [comment]
reopening since related issue has additional issues. Looks like the problem is with how the filepath is parsed, since the closing ] is being considered as part of the path, but the [ isn't. Fix will probably be somewhere in the ADS core code.

**Comment by maccer83** [comment]
Thanks, was just going to say that the other bug wasn't really about the clickable link in the file path :-)

There's a related issue with spaces in folder names (such as the "Stored Procedures" folders created by SSDT) breaking the clickable links, not sure if I should create a new bug report about that?
![Space_in_folder_name_breaks_clickable_link](https://user-images.githubusercontent.com/12064302/114069669-885dc200-989f-11eb-88cf-1ea709b1f464.png)

**Comment by kisantia** [comment]
@maccer83 yes please open a new bug for the spaces breaking the clickable links. Thanks!
